packages/stryker-typescript/package.json

```diff
@@ -51,5 +51,17 @@
   "peerDependencies": {
     "stryker-api": "^0.11.0",
     "typescript": "^2.5.0"
+  },
+  "initStrykerConfig": {
+    "tsconfigFile": "tsconfig.json",
+    "files": [],
+    "coverageAnalysis": "off",
+    "mutator": "typescript",
```

Comment:
Why set this? Shouldn't this be set by default?

Reply:
Ah yes, you are right. I'll remove it.